.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: NuGet Generation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: windows-latest
+    name: Update NuGet
+    steps:
+
+      - name: Checkout repository
+        uses: actions/checkout@master
+
+      - name: Build solution and generate NuGet package
+        run: |
+          cd initialstate.NET
+          dotnet pack -c Release -o out
+
+      - name: Install NuGet client
+        uses: warrenbuckley/Setup-Nuget@v1
+
+      - name: Add private GitHub registry to NuGet
+        run: nuget sources add -name "GitHub" -Source https://nuget.pkg.github.com/tektronix/index.json -Username tektronix -Password ${{ secrets.GitHub_Package_Registry_Key }}
+
+      - name: Push generated package to GitHub registry
+        run: nuget push .\InitialState.NET\InitialState.NET\out\*.nupkg -Source "GitHub" -SkipDuplicate
```

Comment:
@dwyban Can you confirm that this is where the .nupkg file goes when a build is run? Thanks!

Reply:
@zkoppert I've been using nuget.exe to generate and publish the package.  

The command I run on my PC to generate the package is
`
nuget pack -Build -Properties Configuration=Release
`
I run this command from the directory the .csproj is in which is the same folder the .nuspec is in.  The .nupkg that is output ends up in the same directory.  I updated the .gitignore file to ignore .nupkg files so it wouldn't be added to the repo.

To publish I then run
`
nuget push <packagename>.nupkg <API Key> -Source https://api.nuget.org/v3/index.json
`
from my command line.

Reply:
The location the .nupkg files ends up in doesn't seem to matter.  When it is created it contains all the necessary metadata so it can be pushed from any location.